ygen/protogen.go

```diff
@@ -395,6 +522,7 @@ func genProto3Msg(msg *yangDirectory, msgs map[string]*yangDirectory, state *gen
 				annotateSchemaPaths: cfg.annotateSchemaPaths,
 				annotateEnumNames:   cfg.annotateEnumNames,
 				parentPackage:       parentPkg,
+				nestedMessages:      cfg.nestedMessages,
```

Comment:
Enough of cfg seems to be going in to consider just embedding the whole type.

Reply:
Done - and I think this change makes a lot of things neater (reuse of the `protoDefinitionArgs` throughout rather than a lot of repopulation).

Reply:
Yep, good stuff, this is much more readable.